trueline.sh

```diff
@@ -445,6 +445,34 @@ _trueline_cmd_duration_segment() {
     fi
 }
 
+_trueline_distro_icon_segment() {
+    
+    set_distribution_icon() {
+        case "$(uname -a)" in
+            *ubuntu*) distro_icon=" " ;;
+            *fedora*) distro_icon=" " ;;
+            *debian*) distro_icon=" " ;;
+            *arch*) distro_icon=" " ;;
+            *) distro_icon=" " ;;
+        esac
+    }
+
+    # Call the set_distribution_icon function to set the distro_icon.
+    set_distribution_icon
+
+    local some_content="$distro_icon"
+
+    if [[ -n "$some_content" ]]; then
```

Comment:
The set_distribution function always return a non-empty string right? So this guard is useless here.. If this only works for linux then I would change the case above to return an empty string when no known distro is found

Reply:
In fact you can add a default empty distro symbols in the `TRUELINE_SYMBOLS_DEFAULT" array

Reply:
Sounds nice. But I thought instead of an empty string to add a coffee icon. So the user will be able to have this functionality to their machine what so ever. So I removed completely this if statement. Also now I added a case where the user has a linux distribution but there is not listed to the case statement so they will take the default linux icon (penguin) so in this way we cover both.
* If user has a linux distro but there is not listed in the case.
* I user uses the true line in a non linux machine.